raster/r.in.pdal/info.h

```diff
@@ -31,7 +32,7 @@
 #if (PDAL_VERSION_MAJOR >= 2 && PDAL_VERSION_MINOR > 4) || \
     (PDAL_VERSION_MAJOR == 2 && PDAL_VERSION_MINOR == 4 && \
      PDAL_VERSION_PATCH == 3)
-#define PDAL_USE_NOSRS 1
+#define R_IN_R_IN_PDAL_USE_NOSRS 1
```

Comment:
```suggestion
#define R_IN_PDAL_USE_NOSRS 1
```

Thanks for improving this.